py3-distlib.yaml

```diff
@@ -28,31 +34,34 @@ pipeline:
       tag: ${{package.version}}
       expected-commit: ab5f8e797fbc56a0e3488bba68d05e7a602cb63f
 
-  - name: Python Build
-    runs: |
-      python -m build
-      python3 -m installer -d "${{targets.destdir}}" dist/*.whl
-      find ${{targets.destdir}} -name "*.pyc" -exec rm -rf '{}' +
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: python${{range.key}} version of ${{vars.pypi-package}}
+    dependencies:
+      provider-priority: ${{range.value}}
+      provides:
+        - py3-${{vars.pypi-package}}
+    pipeline:
+      - uses: py/build-installer
```

Comment:
took this file as it is right now, and added py3-supported-pip to the envionment.contents.packages list and then it works with 'pip-build-install'.

I suspect the same is true for many of the other ones.

Its annoying that you have to get the dependencies rigth for the pipeline to build.  That is fallout of the pipeline being "multi-version".  It _could_ say it depends on `py3-supported-build`.  maybe that is the rigth way to do things, but that would cause pain for the non-versioned path.... they'd get all the python3-base things installed for a build of just the latest python3.

Reply:
> I suspect the same is true for many of the other ones.
Yeah, I've only encountered one that it didn't work for but encountering that one made me think I was better off using build-installer for things that already specified it and thus I made the pipeline. I'm happy to only change it when something doesn't seem to work with the typical one we use.